ProcessMaker/Policies/ProcessRequestPolicy.php

```diff
@@ -33,29 +33,8 @@ public function before(User $user)
      */
     public function view(User $user, ProcessRequest $processRequest)
     {
-        if ($processRequest->user_id == $user->id) {
-            return true;
-        }
-
-        if ($processRequest->hasUserParticipated($user)) {
-            return true;
-        }
-
-        if ($user->hasPermission('edit-request_data')) {
-            return true;
-        }
-
-        if ($processRequest->canUserClaimASelfServiceTask($user)) {
-            return true;
-        }
-
-        if ($processRequest->processVersion->usersCanEditData()->where('id', $user->id)->exists()) {
-            return true;
-        }
-
-        if ($user->can('view-all_requests')) {
-            return true;
-        }
+        // Policy defined in ForUserScope
+        return $processRequest->forUser($user)->exists();
```

Comment:
@nolanpro 

The query executed  here is:

```sql
SELECT EXISTS 
(
	SELECT * 
	FROM `process_requests` 
	WHERE 
	(
		`user_id` = ? 
		OR 
		(
			`id` IN 
			(
				SELECT `process_request_id` 
				FROM `process_request_tokens` 
				WHERE `user_id` = ?
			)
		) 
		OR 
		(
			(
				`id` IN 
				(
					SELECT `process_request_id` 
					FROM `process_request_tokens` 
					WHERE `is_self_service` = ? 
					AND json_contains (`self_service_groups`, ?, '$."users"')
				) 
				OR `id` IN 
				(
					SELECT `process_request_id` 
					FROM `process_request_tokens` 
					WHERE `is_self_service` = ? 
						AND JSON_OVERLAPS (JSON_EXTRACT (`self_service_groups`,'$."groups"'), ?)
				)
			)
		) 
		OR 
		(
			(
				`process_id` IN 
				(
					SELECT `process_id` 
					FROM `processables` 
					WHERE 
					(
						`processable_type` = ? 
						AND `processable_id` = ? 
						AND `method` = ?
					)
				) 
				OR `process_id` IN 
				(
					SELECT  `process_id` 
					FROM `processables` 
					WHERE (`processable_type` = ? AND `method` = ?) 
					AND `processable_id` IN (?)
				)
			)
		)
    )
) AS `exists` 
```

If the user has any requests, then he will have access to all requests.

**Develop branch**
[develop-branch.webm](https://user-images.githubusercontent.com/16992680/215765922-3733b829-c639-4bed-b4fa-9291e96c69b2.webm)

**bugfix/FOUR-7325 branch**
[bugfix-FOUR-7325.webm](https://user-images.githubusercontent.com/16992680/215765973-e6bf2ff7-1fbd-4805-b2ac-11083b20b052.webm)